enable/abstract_window.py

```diff
@@ -86,22 +86,24 @@ class AbstractWindow(HasTraits):
     resized = Event
 
     # Whether to enable damaged region handling
-    use_damaged_region = Bool(False)
+    use_damaged_region = Bool(False, transient=True)
 
     # The previous component that handled an event.  Used to generate
     # mouse_enter and mouse_leave events.  Right now this can only be
     # None, self.component, or self.overlay.
-    _prev_event_handler = Instance(Component)
+    _prev_event_handler = Instance(Component, transient=True)
 
     # (dx, dy) integer size of the Window.
-    _size = Trait(None, Tuple)
+    _size = Trait(None, Tuple, transient=True)
```

Comment:
I added `transient=True` for all traits that were previously explicitly listed in `__getstate__` to preserve behavior.  Perhaps we should re-evaluate though what actually should/shouldn't be transient